surveys/2025/safety-critical/questions.md

```diff
@@ -0,0 +1,78 @@
+# Rust Language & Ecosystem for Safety-Critical: Project Goals 2026
+
+Hey there! Thanks for taking the time for this survey. Please provide your feedback on those things, which, if available would improve the ability for your company to use the Rust programming language in a safety-critical context. We will use your feedback to inform which Rust Project Goal(s) the Safety-Critical Rust Consortium should propose for 2026.
+
+### Your Safety-Critical industry
+
+Type: free form
+
+### Safety-Critical Specification (ISO 26262, DO-178, etc.)
+
+Type: free form
+
+### Which of the following Rust language features, if available, would improve the ability for you to use the Rust programming language in safety-critical software development?
+
+Type: matrix
+
+Rows:
+
+- Branch Coverage stabilized
+- Modified Condition / Decision Coverage (MC/DC)
+- Debug Info (DWARF) support improved
+- Stack Protector
+- Trust Zone
+- Control Flow Integrity
+- Deterministic Code Generation
+- Coverage Instrumentation for Macros
+- Contracts (pre/post condition)
+
+Columns:
+
+- Would unblock development
+- Would be good to have
+- Not needed for my case
+- Haven't heard of this before
+
+### If there's another feature of <u>the Rust language</u> which would unblock your development, please share it here with the details of what it would unblock.
+
+Type: free form
+
+### Which of the following Rust ecosystem tools/features, if available, would improve the ability for you to use the Rust programming language in safety-critical software development?
+
+Type: matrix
+
+Rows:
+
+- Safety-Qualified Clippy
+- Safety-Qualified Rustfmt
+- Safety-Qualified Test Runner
+- Safety-Certified Async Runtime
+- Safety-Certified libcore
+- Safety-Certified libstd
+
+Columns:
+
+- Would unblock development
+- Would be good to have
+- Not needed for my case
+- Haven't heard of this before
```

Comment:
If you think some readers could not have enough context about an option, would it make sense adding more context to external resources explaining what these items are?